File: Python Excel Examples/PropertiesApiDemo/getDocumentProperty.py
import spirecloudexcel
from spirecloudexcel.configuration import Configuration as ExcelConfiguration
from spirecloudexcel.api.properties_api import PropertiesApi

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = ExcelConfiguration(appId, appKey,baseUrl)
api = spirecloudexcel.api.properties_api.PropertiesApi(configuration)
name = "GetDocumentProperty.xlsx"
propertyName = "Title"
password = ""
folder = "ExcelDocument"
storage = ""
result=api.get_document_property(name, propertyName, password=password, folder=folder, storage=storage)

File: PHP Excel Examples/PropertiesApiDemo/getDocumentProperty.php
<?php

use Spire\Cloud\Excel\Sdk\Api\PropertiesApi;
use Spire\Cloud\Excel\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl = "https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey, $baseUrl);
$apiInstance = new PropertiesApi($configuration);

$name = "GetDocumentProperty_Title.xlsx";
$sourcePath = "input";
$propertyName = "Title";
$password = null;
$storage = null;
$response = $apiInstance->getDocumentProperty($name, $propertyName, $password, $sourcePath, $storage);
